source/specs/config.rst

```diff
@@ -871,6 +871,39 @@ Format::
 
   type : Reserved
 
+Altpage
+^^^^^^^
+
+Alternate page allows a specific configuration item to appear at multiple pages as desired. Lets describe this using an example.
+
+::
+
+  - MEMORY_CFG_DATA :
+
+    - !expand { CFGHDR_TMPL : [ MEMORY_CFG_DATA, 0x200, 0, 0 ] }
```

Comment:
Let us remove this line since it is not related.
Removing it will make it easier.